common-service-core/src/main/java/org/zowe/apiml/product/routing/RoutedServices.java

```diff
@@ -78,7 +78,12 @@ public RoutedService getBestMatchingApiUrl(String serviceUrl) {
                 RoutedService value = serviceEntry.getValue();
                 int size = value.getServiceUrl().length();
                 //Remove last slash for service url
-                String routeServiceUrl = UrlUtils.removeLastSlash(value.getServiceUrl().toLowerCase());
+                String routeServiceUrl;
+                if ("/".equals(value.getServiceUrl())) {
+                    routeServiceUrl = value.getServiceUrl();
```

Comment:
shouldn't be `routeServiceUrl = value.getServiceUrl().toLowerCase();` also here?

Reply:
it is not necessary in this context, if serviceUrl is equals "/" than lowercase "/" is still "/"

Reply:
Ouch! you're right, it was also comparing `getServiceUrl()`. Forget it